pkg/controller/che/create.go

```diff
@@ -341,13 +198,13 @@ func (r *ReconcileChe) CreateSecret(instance *orgv1.CheCluster, m map[string][]b
 	return nil
 }
 
-func (r *ReconcileChe) CreateTLSSecret(instance *orgv1.CheCluster, url string, name string) (err error) {
+func (r *ReconcileChe) CreateTLSSecret(instance *orgv1.CheCluster, url string, name string, clusterAPI deploy.ClusterAPI) (err error) {
```

Comment:
This method create TLS secret with specific purpose and canot be used for any TLS secret creation. The name should reflect that.